builtin/game/item.lua

```diff
@@ -514,7 +514,7 @@ function core.node_dig(pos, node, digger)
 		.. node.name .. " at " .. core.pos_to_string(pos))
 
 	local wielded = digger and digger:get_wielded_item()
-	local drops = core.get_node_drops(node, wielded and wielded:get_name())
+	local drops = core.get_node_drops(node, wielded and wielded:get_name(), wielded, digger)
```

Comment:
```suggestion
	local drops = core.get_node_drops(node, wielded and wielded:get_name(), ItemStack(wielded), digger)
```
should probably do this so the callback does not accidentally modify the wielded item